pkg/internal/checkup/checkup.go

```diff
@@ -47,3 +75,51 @@ func (c *Checkup) Teardown(ctx context.Context) error {
 func (c *Checkup) Results() status.Results {
 	return status.Results{}
 }
+
+func newDPDKVMI(checkupConfig config.Config) *kvcorev1.VirtualMachineInstance {
+	const (
+		CPUSocketsCount   = 1
+		CPUCoresCount     = 4
+		CPUTreadsCount    = 2
+		rootDiskName      = "rootdisk"
+		cloudInitDiskName = "cloudinitdisk"
+		userData          = `#cloud-config
+user: cloud-user
+password: 0tli-pxem-xknu
+chpasswd:
+  expire: false`
+		westNetworkName   = "nic-west"
+		westNICPCIAddress = "0000:06:00.0"
+		eastNetworkName   = "east-west"
+		eastNICPCIAddress = "0000:07:00.0"
+
+		terminationGracePeriodSeconds = 180
+	)
+
+	return vmi.New(randomizeName(VMINamePrefix),
+		vmi.WithoutCRIOCPULoadBalancing(),
+		vmi.WithoutCRIOCPUQuota(),
+		vmi.WithoutCRIOIRQLoadBalancing(),
+		vmi.WithDedicatedCPU(CPUSocketsCount, CPUCoresCount, CPUTreadsCount),
+		vmi.WithSRIOVInterface(westNetworkName, checkupConfig.DPDKWestMacAddress.String(), westNICPCIAddress),
+		vmi.WithMultusNetwork(westNetworkName, checkupConfig.NetworkAttachmentDefinitionName),
+		vmi.WithSRIOVInterface(eastNetworkName, checkupConfig.DPDKEastMacAddress.String(), eastNICPCIAddress),
+		vmi.WithMultusNetwork(eastNetworkName, checkupConfig.NetworkAttachmentDefinitionName),
+		vmi.WithNetworkInterfaceMultiQueue(),
+		vmi.WithRandomNumberGenerator(),
+		vmi.WithHugePages(),
+		vmi.WithMemoryRequest("8Gi"),
+		vmi.WithTerminationGracePeriodSeconds(terminationGracePeriodSeconds),
+		vmi.WithNodeSelector(checkupConfig.DPDKNodeLabelSelector),
+		vmi.WithPVCVolume(rootDiskName, "rhel8-yummy-gorilla"),
```

Comment:
@RamLavi The VMI's PVC is hard coded for now.
We need to think what to do with it.

Reply:
ACK. we can change that once we have a proper PCV